ui.admin-mui/src/components/Settings/Import/nightbot.tsx

```diff
@@ -45,152 +51,90 @@ const PageSettingsModulesImportNightbot: React.FC<{
-              }
-            },
-          );
-        });
-      } catch (error) {
-        console.error('ERROR DURING IMPORT: ', error);
-      }
-    }
-    if (failCount > 0) {
-      enqueueSnackbar(`${failCount} videos failed to import.`, { variant: 'info' });
-    }
-    enqueueSnackbar('Playlist import completed.', { variant: 'success' });
-  };
+  }, [enqueueSnackbar]);
 
   return (
-    <Box ref={ref} id='nightbot'>
-      <Typography variant='h2' sx={{ pb: 2 }}>Nightbot</Typography>
+    <Box ref={ref} id="nightbot">
+      <Typography variant="h2" sx={{ pb: 2 }}>
+        Nightbot
+      </Typography>
       <Paper elevation={1} sx={{ p: 1 }}>
         {userLoadInProgress}
         <TextField
           disabled
           fullWidth
-          variant='filled'
+          variant="filled"
           value={userLoadInProgress ? 'Loading user data...' : user}
           label={translate('integrations.lastfm.settings.username')}
           InputProps={{
-            endAdornment: <InputAdornment position='end'> { user !== 'Not Authorized'
-              ? <Button color='error' variant='contained' onClick={revoke}>Revoke</Button>
-              : <Button color='success' variant='contained' onClick={authorize}>Authorize</Button>
-            }
-            </InputAdornment>,
+            endAdornment: (
+              <InputAdornment position="end">
+                {user !== 'Not Authorized' ? (
+                  <Button color="error" variant="contained" onClick={revoke}>
+                    Revoke
+                  </Button>
+                ) : (
+                  <Button
+                    color="success"
+                    variant="contained"
+                    onClick={authorize}
+                  >
+                    Authorize
+                  </Button>
+                )}
+              </InputAdornment>
+            ),
           }}
         />
-        <Stack direction='row' sx={{
-          pt: 1, textAlign: 'center',
-        }}>
-          <Button sx={{ width: '300px' }} color='primary' variant='contained' disabled={user === 'Not Authorized'} onClick={importPlaylist}>Import playlist</Button>
+        <Stack direction="row" spacing={1}>
+          <Button
+            color="primary"
+            variant="contained"
+            disabled={user === 'Not Authorized'}
+            onClick={() => importPlaylist(accessToken)}
+          >
+            Import playlist
+          </Button>
+          <Button
+            color="primary"
+            variant="contained"
+            disabled={user === 'Not Authorized'}
+            onClick={() => importCustomCommands(accessToken)}
+          >
+            Import commands
+          </Button>
+          <Button
+            color="primary"
+            variant="contained"
+            disabled={user === 'Not Authorized'}
+            onClick={() => importTimers(accessToken)}
+          >
+            Import timers
+          </Button>
         </Stack>
```

Comment:
Thinking of little bit better styling, wdyt?

```suggestion
       <Stack direction="row" spacing={1} sx={{ pt: 1 }}>
          <Button
            fullWidth
            color="primary"
            variant="contained"
            disabled={user === 'Not Authorized'}
            onClick={() => importPlaylist(accessToken)}
          >
            Import playlist
          </Button>
          <Button
            fullWidth
            color="primary"
            variant="contained"
            disabled={user === 'Not Authorized'}
            onClick={() => importCustomCommands(accessToken)}
          >
            Import commands
          </Button>
          <Button
            fullWidth
            color="primary"
            variant="contained"
            disabled={user === 'Not Authorized'}
            onClick={() => importTimers(accessToken)}
          >
            Import timers
          </Button>
        </Stack>
```